contentlayer.config.js

```diff
@@ -28,6 +32,28 @@ const sharedFields = {
 };
 
 const computedFields = {
+  description: {
+    type: "string",
+    /* eslint no-underscore-dangle: off */
+    resolve: async (doc) => {
```

Comment:
Why are we having to do this when this is closed here: https://github.com/datopian/flowershow/issues/297? Is it b/c we are on an old version of flowershow? We are about to upgrade flowershow so personally i wouldn't have this bespoke stuff here and would just fix the descriptions for the posts/podcasts where we need it for now ...